examples/uudecode.c

```diff
@@ -80,6 +80,7 @@ uudecode (const char *progname, int argc, char **argv)
 	int state = 0;
 	mode_t mode;
 	size_t n;
+	int optind = 0;
```

Comment:
The declaration of `optind` will need wrapped in `#ifdef WIN32` or it will conflict with the globally scoped `optind` defined in unistd.h on *nix systems.

Reply:
Ah. Sorry for that. Didn't know it is a system variable on *nix. Fixed. Also changed default value to 1.